lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -27,6 +27,7 @@ $(document).ready(function() {
     const slides = [InstructionSlide]; // ReportbackItemKudosSlide, ContextSlide
 
     $('.chrome').find('> .wrapper').before(`<div id="${jsxElementHook}"></div>`);
+    $('.messages').remove();
```

Comment:
Let's set this to `$('.messages:not(.-error))` so we don't accidentally remove an error message here.